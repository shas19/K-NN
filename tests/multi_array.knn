let t = initArray(4, 5) in 
let t[1,2] = 3.0 in
let b = 5.0 in 
t[1,2]*2.0