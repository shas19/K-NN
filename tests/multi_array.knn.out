<T>
  <k>
    0.60000000000000000e1 ~> .
  </k>
  <env>
    $1 |-> 29
    t |-> 0
  </env>
  <store>
    0 |-> array ( 1 , 4 )
    1 |-> array ( 6 , 5 )
    2 |-> array ( 12 , 5 )
    3 |-> array ( 18 , 5 )
    4 |-> array ( 24 , 5 )
    5 |-> array ( 30 , 5 )
    6 |-> 0.00000000000000000e0
    7 |-> 0.00000000000000000e0
    8 |-> 0.00000000000000000e0
    9 |-> 0.00000000000000000e0
    10 |-> 0.00000000000000000e0
    11 |-> array ( 12 , 5 )
    12 |-> 0.00000000000000000e0
    13 |-> 0.00000000000000000e0
    14 |-> 0.30000000000000000e1
    15 |-> 0.00000000000000000e0
    16 |-> 0.00000000000000000e0
    17 |-> array ( 18 , 5 )
    18 |-> 0.00000000000000000e0
    19 |-> 0.00000000000000000e0
    20 |-> 0.00000000000000000e0
    21 |-> 0.00000000000000000e0
    22 |-> 0.00000000000000000e0
    23 |-> array ( 24 , 5 )
    24 |-> 0.00000000000000000e0
    25 |-> 0.00000000000000000e0
    26 |-> 0.00000000000000000e0
    27 |-> 0.00000000000000000e0
    28 |-> 0.00000000000000000e0
    29 |-> array ( 30 , 5 )
    30 |-> 0.00000000000000000e0
    31 |-> 0.00000000000000000e0
    32 |-> 0.00000000000000000e0
    33 |-> 0.00000000000000000e0
    34 |-> 0.00000000000000000e0
  </store>
  <nextLoc>
    35
  </nextLoc>
  <shape>
    t |-> 4 , 5
  </shape>
</T>
